src/module-elasticsuite-analytics/.gitlab-ci.yml

```diff
@@ -0,0 +1,46 @@
+image: chialab/php:7.1
```

Comment:
This .gitlab-ci.yml file can probably be removed ?

Reply:
Done